### PR TITLE
feat: router + help

### PR DESCRIPTION
**Pull request by reneaaron**

### CODE REVIEW

**Comment by rolznz** [comment]
Some minor styling issues vs the [Figma](https://www.figma.com/file/XYlv0E7SY3R20XHjFcn5kI/Bitcoin-Connect?type=design&node-id=0-1&mode=design&t=OLATSpq2yc36PPLC-0):
image

File: src/components/internal/button.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import {html} from 'lit';
+import {withTwind} from '../twind/withTwind';
+import {LwcElement} from '../lwc-element';
+import {color} from '../css/colors';
+import {gradientText} from '../css/gradientText';
+
+export class Button extends withTwind()(LwcElement) {
```
Comment (rolznz) [inline src/components/internal/button.ts:7]: Cool, I didn't know we can directly override existing HTML components. I can't find any docs for this, did you find any? but seems to work.

File: src/components/internal/button.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        style="
+              background-image: linear-gradient(${color('bg-primary')}, ${color(
+          'bg-primary'
+        )}), linear-gradient(to bottom, ${color('primary')}, ${color(
+          'secondary'
+        )});
+              background-origin: border-box;
+              background-clip: content-box, border-box;"
+      ></div>
+      <span style="${gradientText()}">Disconnect</span>
```
Comment (rolznz) [inline src/components/internal/button.ts:24]: This isn't supposed to be here. But why doesn't it show?

File: src/components/routes.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import {html} from 'lit';
+import './bc-start.js';
```
Comment (rolznz) [inline src/components/routes.ts:2]: help was not imported. How does it work?

**Comment by rolznz** [comment]
image

Incorrect alignment on loading / connected screens

File: src/components/bc-navbar.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          @click=${() => store.getState().setRoute(this.to)}
+        >
+          ${backIcon}
+        </div>
+      </div>
+      <div
+        class="font-sans font-medium"
+        style="color: ${color('text-secondary')}"
+      >
+        ${this.title}
```
Comment (reneaaron) [inline src/components/bc-navbar.ts:33]: Use a different name for this prop?

Heading? pageName?

File: src/components/icons/albyLogo.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import {svg} from 'lit';
+import {color} from '../css/colors';
+
+export const albyLogo = svg`<svg width="49" height="24" viewBox="0 0 49 24" fill="none" xmlns="http://www.w3.org/2000/svg" class="w-16 h-16 -my-4">
```
Comment (reneaaron) [inline src/components/icons/albyLogo.ts:4]: Optimize?

File: src/components/icons/linkIcon.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import {svg} from 'lit';
+import {color} from '../css/colors';
+
+export const linkIcon = svg`<svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg" class="w-5 h-5">
```
Comment (reneaaron) [inline src/components/icons/linkIcon.ts:4]: Square?

Comment (rolznz) [reply src/components/icons/linkIcon.ts]: Moved to https://github.com/getAlby/bitcoin-connect/issues/52

File: src/components/bc-navbar.ts (hunk 4 above)
Comment (rolznz) [reply src/components/bc-navbar.ts]: Changed to heading

**Review by rolznz — APPROVED** [review]
tACK

**Pull request merged**